QuoteOfTheDayQuickStart/README.md

```diff
@@ -0,0 +1,417 @@
+# QuickStart - Quote of the Day app for experimentation on Azure App Configuration
+
+This QuickStart will allow you to create a basic application that shows how two categories of users view different messages alongside a quote of the day. 
```

Comment:
We should share more details of the app, explain what the user is going to do, and what the goal is. Suggest below:

In this quickstart, you create an ASP.NET web app named "Quote of the Day." When the app is loaded, it displays a quote. Users can hit the heart button to like it. To improve user engagement, you want to explore whether a personalized greeting message will increase the number of users who like the quote. Therefore, you create a feature flag in Azure App Configuration with two variants. One shows a standard title, and the other shows a greeting message. With the telemetry support of the Feature Management library, you collect and save events of your user interactions into Application Insights. This allows you to analyze the effectiveness of your experiment.